alerts.md

```diff
@@ -108,11 +184,11 @@ labels:
 	app.kubernetes.io/managed-by: Helm
 app: cost-analyzer
 data:
-	alerts.json: '{"alerts":[{"aggregation":"namespace","filter":"kubecost","ownerContact":"calvin+2@kubecost.com","threshold":0.1,"type":"budget","window":"1d"},{"aggregation":"cluster","filter":"cluster-one","threshold":0.1,"type":"budget","window":"1d"},{"aggregation":"namespace","filter":"*","type":"recurringUpdate","window":"7d"},{"aggregation":"namespace","filter":"kubecost","ownerContact":"calvin+2@kubecost.com","type":"recurringUpdate","window":"7d"}],"enabled":true,"frontendUrl":"http://35.239.230.16:9090","globalAlertEmails":["calvin@kubecost.com"],"slackWebhookUrl":"https://hooks.slack.com/services/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"}'
+	alerts.json: <all-fields-and-values-under-alertConfigs-as-json-string>
 ```
 
 -   Ensure that the json string is successfully mapped to the appropriate configs
 
 Next, confirm that Kubecost product has received configuration data:
 
-- Navigate to your Kubecost UI `/notify.html` as well as the respective `namespace.html?name=<namespace-name>` pages to see that the configured alerts are enabled, budgets are set, and email(s) are set.
+- Visit `<your-kubecost-url>/model/getCustomAlertDiagnostics` to view configuration settings as well as next and last scheduled run times for individual alerts
```

Comment:
Can they not visit the UI anymore? This API does not feel like a good replacement to me.

Reply:
Friendly ping...